subworkflows/local/input_check.nf

```diff
@@ -27,8 +28,10 @@ workflow INPUT_CHECK {
 def create_fastq_channel(LinkedHashMap row) {
     // create meta map
     def meta = [:]
-    meta.id         = row.sample
-    meta.single_end = row.single_end.toBoolean()
+    meta.id             = row.sample
+    meta.single_end     = row.single_end.toBoolean()
+    meta.expected_cells = row.expected_cells != null ? row.expected_cells : null
+    meta.seq_center     = row.seq_center ? row.seq_center : params.seq_center
```

Comment:
Just to check, seq_center is not reuired in the samplesheet (given it's inserted intothe BAM header?)

Reply:
It's not required in the samplesheet. It's an optional annotation in the RG line in the BAM header.